gerousiabot/bot_handlers.py

```diff
@@ -0,0 +1,80 @@
+from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup
+from telegram.ext import CommandHandler, MessageHandler, Filters, CallbackQueryHandler, CallbackContext
+
+
+def get_bot_handlers() -> list:
+    """
+    Gets the list of the handlers to be used by the bot
+    :return: the list of the handlers
+    """
+    start_handler = CommandHandler('start', start)
+    user_query_handler = CommandHandler('chkusers', check_users)
+    button_handler = CallbackQueryHandler(keyboard_button_pressed)
+    unknown_handler = MessageHandler(Filters.command, unknown_command)
+
+    # unknown command handler must be the last added handler or we run into logical errors
+    return [start_handler, button_handler, user_query_handler, unknown_handler]
+
+
+def start_keyboard() -> list:
+    """Initialize the keyboard"""
+    keyboard = [InlineKeyboardButton("Check Users", callback_data='user_check'),
+                InlineKeyboardButton("Blank button", callback_data='blank_button')]
+    return keyboard
+
+
+def keyboard_button_pressed(update: Update, context: CallbackContext) -> None:
+    """Parses the CallbackQuery for keyboard buttons"""
+    query = update.callback_query
+    # CallbackQueries need to be answered, even if no notification to the user is needed
+    # Some clients may have trouble otherwise. See https://core.telegram.org/bots/api#callbackquery
+    query.answer()
+
+    query.edit_message_text(text=f"Selected option: {query.data}")  # for debugging, remove once final
+    if query.data == "user_check":
+        users = format_user_list(get_user_list())
+        context.bot.send_message(chat_id=update.effective_chat.id, text=f"Users currently on the server:\n" + users)
+    elif query.data == "blank_button":
+        context.bot.send_message(chat_id=update.effective_chat.id, text="Nothing to see here.")
+    else:
+        print("Nothing happened.")
+        return
+
+
+def get_user_list() -> list:
+    """
+    Query discord user list.
+    Not yet implemented. Returns test data
+    """
+    user_list = ['rick', 'ash', 'jk']
+    return user_list
+
+
+def format_user_list(user_list: list) -> str:
+    """ Returns a formatted user list with one name per line."""
+    string_layout = str()
+    for name in user_list:
```

Comment:
[Suggestion] This is a good opportunity to use a functional `map` approach, as oppose to the imperative programming style you currently have.

https://realpython.com/python-map-function/

Reply:
Implemented.